assets/compat.ini

```diff
@@ -683,3 +683,13 @@ UCET00844 = true
 UCUS98705 = true
 UCED00971 = true
 UCUS98713 = true
+
+[ForceSoftwareRenderer]
+# Darkstalkers
+ULES00016 = true
+ULUS10005 = true
+
+[DarkStalkersPresentHack]
+# Darkstalkers

```

Comment:
Apparently these lists are missing ULJM05005?

-[Unknown]

Reply:
Ah, guess I missed this one since it's called Vampire Chronicles in certain regions.